test/algorithms/set_operations/intersection/intersection.cpp

```diff
@@ -869,21 +842,6 @@ int test_main(int, char* [])
     test_rational<bg::model::d2::point_xy<boost::rational<int> > >();
 #endif
 
-#if defined(BOOST_GEOMETRY_TEST_FAILURES)
-    // ticket #10868 still fails for 32-bit integers
-    test_ticket_10868<int32_t>("MULTIPOLYGON(((33520458 6878575,33480192 14931538,31446819 18947953,30772384 19615678,30101303 19612322,30114725 16928001,33520458 6878575)))");
-
-#if !defined(BOOST_NO_INT64_T) || defined(BOOST_HAS_MS_INT64)
-    test_ticket_10868<int64_t>("MULTIPOLYGON(((33520458 6878575,33480192 14931538,31446819 18947953,30772384 19615678,30101303 19612322,30114725 16928001,33520458 6878575)))");
-#endif
-
-    if (BOOST_GEOMETRY_CONDITION(sizeof(long) * CHAR_BIT >= 64))
-    {
-        test_ticket_10868<long>("MULTIPOLYGON(((33520458 6878575,33480192 14931538,31446819 18947953,30772384 19615678,30101303 19612322,30114725 16928001,33520458 6878575)))");
-    }
-
-    test_ticket_10868<long long>("MULTIPOLYGON(((33520458 6878575,33480192 14931538,31446819 18947953,30772384 19615678,30101303 19612322,30114725 16928001,33520458 6878575)))");
-#endif
```

Comment:
Moved to `intersection_integer.cpp` - but it was excluded anyway, and still fails, for reasons not investigated.
Apart from that, verifying WKT expectations is cumbersome and normally not done in these tests.